packages/helpers/src/customPropTypes.js

```diff
@@ -37,3 +38,5 @@ export const InputValidTypes = {
 };
 
 InputValidTypes.ALL = Object.values(InputValidTypes);
+
+export const RefPropType = PropTypes.shape({ current: PropTypes.instanceOf(Object) });
```

Comment:
If this is a generic prop type for a ref, then this can be parameterized with the type of what goes under `current`. (also `PropType` in the name is redundant)
Like 
```js
export const RefOf = (type) => PropTypes.shape({ current: PropTypes.instanceOf(type) });
```

Reply:
```
PropTypes.oneOfType([
    PropTypes.func, 
    PropTypes.shape({ current: PropTypes.any })
])
```